From 114e9bbf4176ad7aaef676cce58fe7752ff28edb Mon Sep 17 00:00:00 2001
From: INAGAKI Hiroshi <musashino.open@gmail.com>
Date: Tue, 12 Sep 2017 14:01:33 +0900
Subject: [PATCH 1/8] make: Add options to BUILD_LDFLAGS

Signed-off-by: INAGAKI Hiroshi <musashino.open@gmail.com>
---
 Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 14cef7a..32a13ea 100644
--- a/Makefile
+++ b/Makefile
@@ -17,7 +17,8 @@ BUILD_LDFLAGS := "\
 	  -X main.version=$(VERSION) \
 	  -X main.gitcommit=$(CURRENT_REVISION) \
 	  -X github.com/mackerelio/mackerel-agent/config.agentName=$(MACKEREL_AGENT_NAME) \
-	  -X github.com/mackerelio/mackerel-agent/config.apibase=$(MACKEREL_API_BASE)"
+	  -X github.com/mackerelio/mackerel-agent/config.apibase=$(MACKEREL_API_BASE) \
+	  -s -w"
 
 .PHONY: all
 all: clean test build
-- 
2.17.1.windows.2

