From 99f1668029f5fa90a7b11068360cbad7e85e0ab9 Mon Sep 17 00:00:00 2001
From: INAGAKI Hiroshi <musashino.open@gmail.com>
Date: Sun, 23 Jun 2019 01:04:48 +0900
Subject: [PATCH 2/7] version: use "-OpenWrt" suffix for version string

---
 version.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/version.go b/version.go
index fa5be66..3cdbf0d 100644
--- a/version.go
+++ b/version.go
@@ -1,5 +1,5 @@
 package main
 
-const version = "0.71.0"
+const version = "0.71.0-OpenWrt"
 
 var gitcommit string
-- 
2.29.2.windows.3

