From 82721d3b74f3477414b91a520e02c0d89ba4a7aa Mon Sep 17 00:00:00 2001
From: INAGAKI Hiroshi <musashino.open@gmail.com>
Date: Sun, 23 Jun 2019 01:04:48 +0900
Subject: [PATCH 2/7] version: use "-OpenWrt" suffix for version string

---
 version.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/version.go b/version.go
index a9d1800..9426eaf 100644
--- a/version.go
+++ b/version.go
@@ -1,5 +1,5 @@
 package main
 
-const version = "0.71.1"
+const version = "0.71.1-OpenWrt"
 
 var gitcommit string
-- 
2.31.1.windows.1

