From 406368f413055c3383f5dfc0ea722067de8d7fed Mon Sep 17 00:00:00 2001
From: INAGAKI Hiroshi <musashino.open@gmail.com>
Date: Fri, 16 Mar 2018 14:26:44 +0900
Subject: [PATCH] build: add -s -w option into ldflags

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 3e32b0f..638c824 100644
--- a/Makefile
+++ b/Makefile
@@ -19,7 +19,7 @@ test: test-deps
 	go test -v ./...
 
 build: deps
-	go build -ldflags "-X main.gitcommit=$(CURRENT_REVISION)" -o $(BIN) .
+	go build -ldflags "-X main.gitcommit=$(CURRENT_REVISION) -s -w" -o $(BIN) .
 
 lint: test-deps
 	go vet ./...
-- 
2.16.1.windows.4

